vulnerabilities.yaml

```diff
@@ -21,6 +21,25 @@
 #
 # Other tag: "ignore-python3: true", used for modules removed from Python 3
 
+- name: "bpo-35755: Remove current directory from posixpath.defpath"
```

Comment:
Honestly, I'm not sure that it's really a security issue. If you consider that it's the case, please open a discussion on security-sig@python.org mailing list.

Reply:
I just added it since it was tagged as security in GitHub and bpo. I have removed it.